scripts/ci/notification/matrix_notification.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+set -e
+
+SCRIPT_DIR="$(dirname "$0")"
+
+fail() {
+    printf '%s\n' "$1" >&2
+    exit 1
+}
+
+notify() {
+    # Prevent eval over unexpected environment variable values
+    BRANCH="$(sanitize_variable "$BRANCH")"
+    TAG="$(sanitize_variable "$TAG")"
+    BUILD_URL="$(sanitize_variable "$BUILD_URL")"
+    BUILD_NUM="$(sanitize_variable "$BUILD_NUM")"
+
+    # FIXME: Checking $CIRCLE_PULL_REQUEST would be better than hard-coding branch names
```

Comment:
I kept this comment, since I still need to investigate if the statement still holds.

Reply:
Any update?